reference/docs/conversion/sakai_11_mysql_conversion.sql

```diff
@@ -765,7 +765,7 @@ ALTER TABLE lti_content MODIFY title VARCHAR(1024);
 ALTER TABLE lti_content MODIFY pagetitle VARCHAR(1024);
 ALTER TABLE lti_content MODIFY consumerkey VARCHAR(1024);
 ALTER TABLE lti_content MODIFY secret VARCHAR(1024);
-alter table lti_content modify custom varchar(65536);
+ALTER TABLE lti_content MODIFY custom MEDIUMTEXT;
```

Comment:
I think I just picked mediumtext to be on the safe side because all but one of the other larger-than-VARCHAR(1024) LTI-columns were using mediumtext.
Well, and because, technically, the original  65536 bytes were one byte more than mysql  TEXT's maximum size. 

..and, obviously, because I don't have a clue about LTI and what that column is for.